src/Plugins/ComplexCore/src/ComplexCore/ComplexCoreLegacyUUIDMapping.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <map>
+#include <string>
+// clang-format off
+namespace complex
+{
+  // Regex to grep UUIDs : {\"([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\",\s\"([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}(?:\s,\s[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})*)\"},\s\/\/\s(\w+)
+  static const std::map<std::string, std::string> k_SIMPL_to_ComplexCore
```

Comment:
I would make these a `std::map<Uuid, Uuid>` instead. And maybe we should consider a bidirectional map (or approximation of one) so we don't have to duplicate the mapping.

Also I think we should use the `FilterTraits<T>::uuid` here to avoid duplicating the uuid definitions.

Reply:
On the topic of bidirectional mapping I think that in this case I could do the same with two std::vectors that are equivalent by index. There are only 450 filter UUIDs and about 3 instances of multiple UUIDs. If it was imlemented this way we could make them std::vector<complex::Uuid> because it wouldn't require each key to be unique. Using some index tricks multiple SIMPL UUIDs could be pulled by searching for duplicate instances in the complex vector...but this introduces the chance for unwanted duplicates [especially since they are UUID version 4 (puesdo-random)].